frontend/src/components/carousel.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
```

Comment:
This was moved to `components/home/carousel.jsx` during the re-org and should be deleted.